packages/esm-pharmacy-app/src/pharmacy-metrics/pharmacy-metrics.scss

```
@@ -0,0 +1,11 @@
+@use '@carbon/styles/scss/spacing';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.cardContainer {
+  background-color: $ui-02;
```

```suggestion
  background-color: colors$.color-01;
```
prefer the colors import from carbon